src/components/editor/editor-screen-table.tsx

```diff
@@ -103,20 +103,26 @@ const EditorScreenTable: FC<Props> = ({
    */
   const buildVersionRows = (metaform: Metaform): MetaformVersionRow[] => {
     const versions = metaformVersions.filter(x => x.data.id as unknown === metaform.id);
+    versions.sort((a, b) => {
+      return a.modifiedAt!.getTime() > b.modifiedAt!.getTime() ? -1 : 1;
```

Comment:
Dangerous to use null assertion, maybe filter first?

Reply:
Good idea.